### PR TITLE
[DetailsList] Adding aria-describedby optional callback for each item

### LINKED ISSUE #3757: [Accessibility] [DetailsList] Feature request for getting getRowAriaDescribedBy
**Problem**: When using DetailsList its a great functionality that we are able to set ariaLabel using getRowAriaLabel, however for non-intuitive tables and dynamic information there is a requirement to get something like 
```getRowAriaDescribedBy``` which could be again like an optional callback to get the aria-descibedBy for each item. 

_Yes_ I am willing to raise a pull-Request, please let me know if there is an already Work In Progress for this or you may suggest any alternate approach/option to achieve this.

### PR DESCRIPTION
**Pull request by akashperfect**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #3757
- [x] Include a change request file using `$ npm run change`

#### Description of changes
Reason: Accessibility
- Adding optional callback aria-describedby for each item in DetailsList.
- Subsequently adding the optional prop to DetailsRow
- Calling the callback if defined in DetailsRow
- Passing the (space separated strings) IDs of elements thus obtained to FocusZone

#### Focus areas to test

Aria-describedby can be tested with Edge + Narrator, when specified on a row

### CODE REVIEW

**Comment by Jahnp** [comment]
@cschleiden thoughts? This looks like a reasonable change to me, but I'm not the best authority on DetailsList.

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx [hunk 1]
```diff
@@ -198,6 +201,7 @@ export class DetailsRow extends BaseComponent<IDetailsRowProps, IDetailsRowState
         componentRef={ this._resolveRef('_focusZone') }
         role='row'
         aria-label={ ariaLabel }
+        ariaDescribedBy={ ariaDescribedBy }
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx]: `aria-describedby` for consistency?

Comment (akashperfect) [reply packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx]: @cschleiden The focusZone for which we are adding the props here has ```ariaDescribedBy``` for aria-describedby. Hence, directly adding here instead, leveraged the property already part of FocusZone component

Comment (cschleiden) [reply packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx]: okay.

**Comment by akashperfect** [comment]
Hi, is there any reactionary work left from my end to get this pull request merged?

**Comment by cschleiden** [comment]
@akashperfect There are conflicts, you have to resolve them before the PR can be merged. 

image

**Comment by akashperfect** [comment]
Hi, resolved the conflicts with this commit 2427e6b. Please let me know if there is anything else needed to be done here.

**Pull request merged**